scaleway/testdata/tem-domain-validation-basic.cassette.yaml

```diff
@@ -0,0 +1,39 @@
+---
+version: 1
+interactions:
+- request:
+    body: '{"project_id":"46fd79d8-1a35-4548-bfb8-03df51a0ebae","domain_name":"terraform-rs.test.local","accept_tos":true}'
+    form: {}
+    headers:
+      Content-Type:
+      - application/json
+      User-Agent:
+      - scaleway-sdk-go/v1.0.0-beta.7+dev (go1.21.5; darwin; amd64) terraform-provider/develop
+        terraform/terraform-tests
+    url: https://api.scaleway.com/transactional-email/v1alpha1/regions/fr-par/domains
+    method: POST
+  response:
+    body: '{"details":[{"current":3,"quota":3,"resource":"TemDomains"}],"message":"quota(s)
```

Comment:
Not sure this is the message we are looking for

Reply:
This file shouldn't have been pushed; it's an old test.